_data/toc.yaml

```diff
@@ -1418,9 +1454,6 @@ manuals:
         path: /ee/admin/backup/back-up-ucp/
       - title: Restore UCP
         path: /ee/admin/restore/restore-ucp/
```

Comment:
Is it possible to change the structure here to improve the URLs for these topics? For example, the other sections are under /ee/ucp/admin. The back up and restore sections are under ee/admin/backup/ and ee/admin/restore. 

It would add consistency to have these under ee/ucp/admin/backup and ee/ucp/admin restore.

Reply:
I'll take this offline with you.